src/site/constants/content-modeling.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Represents the possible values for the "entityBundle" field
+ * of Drupal (or generated) nodes. These values should align with
+ * with the names of the files in the "src/site/layouts directory.
+ */
+const ENTITY_BUNDLES = {
+  BASIC_LANDING_PAGE: 'basic_landing_page',
+  BIOS_PAGE: 'bios_page',
+  CHECKLIST: 'checklist',
+  EVENT_LISTING: 'event_listing',
+  EVENT: 'event',
+  EVENTS_PAGE: 'events_page',
+  FAQ_MULTIPLE_Q_A: 'faq_multiple_q_a',
+  FULL_WIDTH_BANNER_ALERT: 'full_width_banner_alert',
+  HEALTH_CARE_FACILITY_STATUS: 'health_care_facility_status',
+  HEALTH_CARE_LOCAL_FACILITY: 'health_care_local_facility',
+  HEALTH_CARE_LOCAL_HEALTH_SERVICE: 'health_care_local_health_service',
+  HEALTH_CARE_REGION_DETAIL_PAGE: 'health_care_region_detail_page',
+  HEALTH_CARE_REGION_LOCATIONS_PAGE: 'health_care_region_locations_page',
+  HEALTH_CARE_REGION_PAGE: 'health_care_region_page',
+  HEALTH_SERVICES_LISTING: 'health_services_listing',
+  HOME: 'home',
+  LANDING_PAGE: 'landing_page',
+  LEADERSHIP_LISTING: 'leadership_listing',
+  LOCATIONS_LISTING: 'locations_listing',
+  MEDIA_LIST_IMAGES: 'media_list_images',
+  MEDIA_LIST_VIDEOS: 'media_list_videos',
+  NEWS_STORIES_PAGE: 'news_stories_page',
+  NEWS_STORY: 'news_story',
+  OFFICE: 'office',
+  OUTREACH_ASSET: 'outreach_asset',
+  PAGE: 'page',
+  PERSON_PROFILE: 'person_profile',
+  PRESS_RELEASE: 'press_release',
+  PRESS_RELEASES_LISTING: 'press_releases_listing',
+  PRESS_RELEASES_PAGE: 'press_releases_page',
+  PUBLICATION_LISTING: 'publication_listing',
+  Q_A: 'q_a',
+  REGIONAL_HEALTH_CARE_SERVICE_DES: 'regional_health_care_service_des',
+  STEP_BY_STEP: 'step_by_step',
+  STORY_LISTING: 'story_listing',
+  SUPPORT_RESOURCES_ARTICLE_LISTING: 'support_resources_article_listing',
+  SUPPORT_RESOURCES_DETAIL_PAGE: 'support_resources_detail_page',
+  SUPPORT_SERVICE: 'support_service',
+  VA_FORM: 'va_form',
+  VAMC_OPERATING_STATUS_AND_ALERTS: 'vamc_operating_status_and_alerts',
+};
+
+module.exports = {
+  ENTITY_BUNDLES,
+};
```

Comment:
IDK about this file name but I am a little tired of magic strings floating around to represent entityBundles

Reply:
🧙 lol